src/test/java/com/edmunds/rest/databricks/fixtures/DatabricksFixtures.java

```diff
@@ -146,7 +146,12 @@ public static HttpServiceUnavailableRetryStrategy getHttpServiceUnavailableRetry
 
   public static DatabricksServiceFactory getDatabricksServiceFactory() {
     if (factory == null) {
-      factory = DatabricksServiceFactory.Builder.createTokenAuthentication(TOKEN, HOSTNAME).build();
+      factory = DatabricksServiceFactory.Builder.createTokenAuthentication(TOKEN, HOSTNAME)
+              .withMaxRetries(3)
+              .withSoTimeout(60*1000)
```

Comment:
We should use Duration or some other class to capture time as a discrete value. e.g. Duration.ofMinutes(1).toMillis()

Reply:
+1. 
Note: filtering operations seems to take forever (for the moment), that's why the large timeout value (1 minute). I understood that's a known problem and it will be addressed soon.